### ISSUE TITLE
Remove Basemap and GEO.ca surveys from GEO.ca

### ISSUE THREAD

**Issue by Andrea-Merry** [Canadian-Geospatial-Platform/geo.ca#97]
Please remove both surveys from GEO.ca on December 22nd, 2023.
Thank you,
Andrea

**Comment by jvanulde** [comment]
@indiciumx will you have the necessary access on the 22nd to do this? If not please provide @PVautour with instructions. @elatsis please ensure this gets completed. Merci.

**Comment by bo-lu** [comment]
Committed to dev and stage. Prod is pending

For posterity, I removed these lines of html:

```
<div class="elementor-text-editor elementor-clearfix"><p style="text-align: center;"><a href="https://www.surveymonkey.ca/r/2MW7KJK"><span style="color: #ffffff;"><b>Your feedback is important to us. In order to improve our platform and your experience using it, <u>we invite you to complete a short 10 minute survey</u>.</b></span></a></p></div>

<div class="elementor-text-editor elementor-clearfix"><p style="text-align: center;"><a href="https://www.surveymonkey.ca/r/2MW7KJK?lang=fr"><span style="color: #ffffff;">Vos commentaires sont importants pour nous. Afin d&rsquo;améliorer notre plateforme et votre expérience d&rsquo;utilisation, <u>nous vous invitons à répondre à un court sondage de 10 minutes.</u></span></a></p></div>

<div class="elementor-text-editor elementor-clearfix"><p style="text-align: center;"><strong><span style="color: #ffffff;">We need your feedback! Please take our <a style="color: #ffffff;" href="https://www.surveymonkey.com/r/KMH8JWN?site=geo">survey of the Canadian Basemap.</a></span></strong></p></div>

<div class="elementor-text-editor elementor-clearfix"><p style="text-align: center;"><span style="color: #ffffff;">Nous avons besoin de vos commentaires! Veuillez répondre à notre <a style="color: #ffffff;" href="https://www.surveymonkey.com/r/KMH8JWN?site=[geo]&amp;lang=fr">sondage sur la carte de base du Canada.</a></span></p></div>
```

**Comment by jvanulde** [comment]
@Andrea-Merry please review on staging: https://stage.geo.ca/

**Comment by bo-lu** [comment]
This can be closed with commit: d35640c4698cd67fe7457fea4597816bf682fca8

**Comment by Andrea-Merry** [comment]
Looks good. Thank you.
